Ports/halflife/package.sh

```diff
@@ -59,7 +77,10 @@ post_install() {
 export LD_LIBRARY_PATH=/home/anon/Games/halflife/
 scriptdir=$(dirname "$0")
 cd $scriptdir
-./xash3d
+./xash3d -console
 EOF
     chmod +x ${SERENITY_INSTALL_ROOT}/home/anon/Games/halflife/hl.sh
+
+    echo "Remember to copy the 'valve/' from your half life installation"
```

Comment:
"Half-Life installation" is probably better capitalization here.

Reply:
Yeah, will make this message nicer.